kqueen/engines/openstack_kubespray.py

```diff
@@ -597,8 +594,9 @@ def _run_ansible(self, inventory="hosts.json", playbook="cluster.yml"):
         args = [
             config.KS_ANSIBLE_PLAYBOOK_CMD, "-b", "-i",
             inventory, playbook,
-            "--extra-vars", "delete_nodes_confirmation=yes",
-            "--extra-vars", "docker_dns_servers_strict=no",
+            "-e", "delete_nodes_confirmation=yes",
```

Comment:
why we need this? seems it can break deprovision actions

Reply:
How? Here I've just add "ansible_python_interpreter=/usr/bin/python3" and use '-e' instead of '--extra-vars' just to make it more compact